src/renderers/native-rt/ReactNativeRTFiberRenderer.js

```diff
@@ -25,6 +25,26 @@ export type TextInstance = number;
 
 const {precacheFiberNode, updateFiberProps} = ReactNativeRTComponentTree;
 
+function processProps(instance: number, props: Props): Object {
+  const propsPayload = {};
+  for (var key in props) {
+    if (key === 'children') {
+      // Skip special case.
+      continue;
+    }
+    var value = props[key];
+    if (typeof value === 'function') {
+      value = {
+        style: 'rt-event',
+        event: key,
```

Comment:
nit: Might it be confusing that `event` ends up holding values like `onTopChange` instead of `topChange`? Maybe we should name it `key` or something instead?

Reply:
Given that this bypasses the event system it won't use the `topChange` mechanism. So not sure if it's confusing if you never new about the previous event system.

Reply:
Sure. Maybe that was just a poor example. My point was that the prop (`key`) usually corresponds to the event _handler_ name rather than the _event_ name. But maybe this new renderer is different?

Reply:
Right. This name is basically only here so that they can name the event name the same as the handler, so that we don't need any mapping logic on our side.

This is just one possible approach though.

Reply:
Okay. Sounds like you've thought about this and it's intentional then so...cool. 😄  Thanks for elaborating.